docs/2.7.0/docs/deploy-daml/infrastructure-architecture/high-availability/basic-ha/domain.rst

```diff
@@ -38,18 +38,18 @@ The sequencer relies on the database for both HA and consistency. The database e
 Many sequencer nodes can be deployed. Each node has concurrent read and write components when accessing the database. The load balancer evenly distributes requests between sequencer nodes.
 
 .. NOTE::
-   The system halts requests to a node whose status is reported as unhealthy by the health endpoint.
+   The system stops sending requests to an unhealthy node.
 
-Ensuring Consistency
-````````````````````
+Consistency and the Database Sequencer
+``````````````````````````````````````
```

Comment:
how about making this one and the next a subsection of Database Sequencer?

Reply:
I'll put this in the next PR with the updated diagrams.